spec/interactors/find_user_by_reset_token_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe FindUserByResetToken do
+  describe ".call" do
+    let(:token) { Encryptor.generate_token }
+
+    context "when successful" do
+      before do
+        @user = create(:confirmed_user, reset_digest: token[0])
+      end
```

Comment:
@Enriikke rubocop doesn't like this but I need the `user` to be instantiated before the subject is called. Using an instance variable seemed like the cleaned way to make this work.